composer.json

```diff
@@ -1,15 +1,24 @@
 {
-    "name": "nsire/badaso-lms-theme",
-    "autoload": {
-        "psr-4": {
-            "Nsire\\BadasoLmsTheme\\": "src/"
-        }
+  "name": "badaso/lms-theme",
+  "authors": [
+    {
+      "name": "nofamex",
+      "email": "nsiren517@gmail.com"
```

Comment:
@nofamex changed this one to uasoft's email in this PR https://github.com/uasoft-indonesia/badaso-lms-module/pull/11 . Do we need to change this one as well or just leave it like this?